app/src/main-process/app-window.ts

```diff
@@ -317,16 +317,23 @@ export class AppWindow {
     this.window.destroy()
   }
 
+  /**
+   * Method to show the save dialog and return the first file path it returns.
+   */
+  public async showSaveDialog(options: Electron.SaveDialogOptions) {
+    const { canceled, filePath } = await dialog.showSaveDialog(
+      this.window,
+      options
+    )
+
+    return !canceled ? filePath ?? null : null
```

Comment:
The docs for this method says that `filePath` will be undefined if the dialog was cancelled and it doesn't look like we actually care about the cancelled bool. Could we simplify this (I'm suggesting this knowing that you're refactoring and not changing legacy behavior, nevertheless it'd be a nice change I think)

```suggestion
    const { filePath } = await dialog.showSaveDialog(this.window, options)
    return filePath ?? null
```

Reply:
Sounds good to me!